installation/public-docker-image-setup.md

```diff
@@ -191,3 +191,4 @@ docker ps -f "status=exited"
 
 #### Prune entire Docker system
 docker system prune
+docker volume prune
```

Comment:
Can we add a sentence about when pruning is required? Or specific errors/behaviors that would lead a user to prune.

Reply:
Not sure about when it is required. In my case, pruning the volume in addition to system prune and trying again worked. I am assuming the volume was corrupted but I could be wrong. @medic/site-reliability-engineering any ideas?

Reply:
Maybe just say something like "Use this prune command when unable to launch the container, or when you'd like to restart the container from a clean state." Based on this documentation https://docs.docker.com/engine/reference/commandline/system_prune/ the single command `docker system prune --volumes` is equivalent to running both the commands above.

Reply:
@kennsippell done if you could give it a second look